backend/src/test/java/gov/cdc/usds/simplereport/service/TestOrderServiceTest.java

```diff
@@ -151,7 +155,41 @@ void fetchTestResults_entryOnlyUser_error() {
 
         // https://github.com/CDCgov/prime-simplereport/issues/677
         // assertSecurityError(() ->
-        // _service.getTestResults(facility.getInternalId().toString()));
-        assertSecurityError(() -> _service.getTestResults(p));
+        // _service.getTestResults(facility.getInternalId()));
+        // assertSecurityError(() -> _service.getTestResults(p));
+    }
+
+    @Test
+    void correctionsTest() {
+        Organization org = _organizationService.getCurrentOrganization();
+        Facility facility = _organizationService.getFacilities(org).get(0);
+        Person p = _dataFactory.createFullPerson(org);
+        TestEvent _e = _dataFactory.createTestEvent(p, facility);
+        TestOrder _o = _e.getTestOrder();
+
+        String reasonMsg = "Testing correction marking as error " + LocalDateTime.now().toString();
+        TestEvent deleteMarkerEvent = _service.correctTestMarkAsError(_e.getInternalId(), reasonMsg);
+        assertNotNull(deleteMarkerEvent);
+
+        assertEquals(TestCorrectionStatus.REMOVED, deleteMarkerEvent.getCorrectionStatus());
+        assertEquals(reasonMsg, deleteMarkerEvent.getReasonForCorrection());
+
+        assertEquals(_e.getTestOrder().getInternalId().toString(), _e.getTestOrderId().toString());
```

Comment:
No need to compare the stringification of the UUIDs, just compare them directly.